diagnostic.md

```diff
@@ -27,18 +27,22 @@ on the body element. Use that variable to increase font-size of `h1`
 elements by 30%.
 
 ```scss
-// your response here
+h1 {
+  $base-font-size: ($font-size * 1.667);
```

Comment:
No, sorry - you have these reversed.
```
h1 {
  font-size: ($base-font-size * 1.3);
}
```
You can vocalize this as 'set font size of H1 to base font size times 1.3'.